src/library/dao/playlistdao.cpp

```diff
@@ -516,7 +526,11 @@ void PlaylistDAO::removeTracksFromPlaylistInner(int playlistId, int position) {
     }
 
     m_playlistsTrackIsIn.remove(trackId, playlistId);
+
     emit trackRemoved(playlistId, trackId, position);
+    if (getHiddenType(playlistId) == PLHT_SET_LOG) {
```

Comment:
note that deleted playlists have type PlaylistDAO::PLHT_UNKNOWN.  This messed me up when I was creating my patch.  If there are any situations where you need to check for deleted playlists as well as existing ones, also check for UNKNOWN.

Reply:
Why should we care about any deleted files?? The play count only affects history playlists.

Reply:
not deleted files, deleted playlists.  At least in my code, if I delete a History playlist, I need to notice that so I can reprocess the last-played-times.  Yours may work differently, though.  As long as the column updates correctly if the user deletes a history playlist, you're fine.

Reply:
Only playlists of type PLHT_SET_LOG affect the *last played* property. I don't understand what you are asking for.

Reply:
The signal tracksRemovedFromPlayedHistory() only considers PLHT_SET_LOG as the naming suggests. Nevertheless I forgot to add the `emit` keyword here.